### PR TITLE
Add Area model

### PR DESCRIPTION
**Pull request by draffensperger**
@Omicron7 I was thinking it might be helpful to add these now so that your work on the endpoint can use the models.

### CODE REVIEW

**Comment by draffensperger** [comment]
As per a chat conversation I had with Brian, a ministry can only be in one area (with one exception that we choose to ignore in our apps), so I'm switching the relationship to a `belongs_to` between `Ministry` and `Area`.

File: app/models/ministry.rb [hunk 1]
```diff
@@ -28,8 +28,7 @@ class Ministry < ActiveRecord::Base
 
   has_many :user_content_locales, dependent: :destroy
   has_many :stories, dependent: :destroy
-  has_many :ministry_areas, dependent: :destroy
-  has_many :areas, through: :ministry_areas
+  belongs_to :area, dependent: :destroy
```
Comment (Omicron7) [inline app/models/ministry.rb]: Remove the dependent destroy, from rails guides:
"You should not specify this option on a belongs_to association that is connected with a has_many association on the other class. Doing so can lead to orphaned records in your database."

Comment (draffensperger) [inline app/models/ministry.rb]: Good catch and that's good to know about! I'll remove it. I think it got leftover from when I had it as `has_many` before.

**Comment by Omicron7** [comment]
:+1:

**Pull request merged**